contracts/src/utils/testing.cairo

```diff
@@ -41,61 +58,59 @@ use eternum::systems::config::labor_config::SetLaborConfig;
-    components.append(QuantityTrackerComponent::TEST_CLASS_HASH);
-    components.append(PositionComponent::TEST_CLASS_HASH);
-    components.append(CapacityComponent::TEST_CLASS_HASH);
-    components.append(ArrivalTimeComponent::TEST_CLASS_HASH);
-    components.append(CaravanMembersComponent::TEST_CLASS_HASH);
-    components.append(CaravanComponent::TEST_CLASS_HASH);
-    components.append(ForeignKeyComponent::TEST_CLASS_HASH);
-    components.append(TradeComponent::TEST_CLASS_HASH);
-    components.append(FungibleEntitiesComponent::TEST_CLASS_HASH);
-    components.append(OrderResourceComponent::TEST_CLASS_HASH);
-    components.append(ResourceComponent::TEST_CLASS_HASH);
-    components.append(StatusComponent::TEST_CLASS_HASH);
-    components.append(AuthRoleComponent::TEST_CLASS_HASH);
-    components.append(AuthStatusComponent::TEST_CLASS_HASH);
-    components.append(TokenApprovalComponent::TEST_CLASS_HASH);
-    components.append(BalanceComponent::TEST_CLASS_HASH);
-    components.append(AgeComponent::TEST_CLASS_HASH);
-    components.append(TravelConfigComponent::TEST_CLASS_HASH);
-    components.append(LaborComponent::TEST_CLASS_HASH);
-    components.append(LaborConfigComponent::TEST_CLASS_HASH);
-    components.append(LaborCostAmountComponent::TEST_CLASS_HASH);
-    components.append(LaborCostResourcesComponent::TEST_CLASS_HASH);
-    components.append(VaultComponent::TEST_CLASS_HASH);
-    components.append(WeightConfigComponent::TEST_CLASS_HASH);
+    components.append(owner::TEST_CLASS_HASH);
+    components.append(movable::TEST_CLASS_HASH);
+    components.append(quantity::TEST_CLASS_HASH);
+    components.append(realm::TEST_CLASS_HASH);
+    components.append(speed_config::TEST_CLASS_HASH);
+    components.append(capacity_config::TEST_CLASS_HASH);
+    components.append(world_config::TEST_CLASS_HASH);
+    components.append(meta_data::TEST_CLASS_HASH);
+    components.append(quantity_tracker::TEST_CLASS_HASH);
+    components.append(position::TEST_CLASS_HASH);
+    components.append(capacity::TEST_CLASS_HASH);
+    components.append(arrival_time::TEST_CLASS_HASH);
+    components.append(caravan_members::TEST_CLASS_HASH);
+    components.append(caravan::TEST_CLASS_HASH);
+    components.append(foreign_key::TEST_CLASS_HASH);
+    components.append(trade::TEST_CLASS_HASH);
+    components.append(fungible_entities::TEST_CLASS_HASH);
+    components.append(order_resource::TEST_CLASS_HASH);
+    components.append(resource::TEST_CLASS_HASH);
+    components.append(status::TEST_CLASS_HASH);
+    components.append(age::TEST_CLASS_HASH);
+    components.append(travel_config::TEST_CLASS_HASH);
+    components.append(labor::TEST_CLASS_HASH);
+    components.append(labor_config::TEST_CLASS_HASH);
+    components.append(labor_cost_amount::TEST_CLASS_HASH);
+    components.append(labor_cost_resources::TEST_CLASS_HASH);
+    components.append(vault::TEST_CLASS_HASH);
+    components.append(weight_config::TEST_CLASS_HASH);
```

Comment:
This would be a go time todo `let mut components = array![weight_config::TEST_CLASS_HASH, ...]`